model/inp/ww3_gint.inp

```diff
@@ -28,7 +28,10 @@ $ Upon execution of the code an out_grd.grd4 is generated via
 $ interpolation of output fields from the various out_grd.grdN
 $ (N varying from 1 to 3) files.
 $
-$ Choice of no extrapolation [0] or to extrapolate [1] with wet points [2]
+$ Interpolation scheme
+$   0: Linear
```

Comment:
Thanks for adding the update here and in the manual! This is exactly what I was mentioning. 

I have one question though, based on what I've looked at the code and what was there before and that we've only added 1 new option (option 2) I thought the options would be more described like: 

```
0: linear interpolation, no extrapolation 
1: linear interpolation, extrapolate with wet points 
2: nearest neighbor interpolation 
```

Or am I misunderstanding?

Reply:
From what I gathered, 1 only works for unstructured grids. I did not check whether that uses some sort of linear interpolation (baricentric coordinates?)

Reply:
Yes, it does.

Reply:
I don't see where 1 only works for unstructured grids?   I can't find anywhere where it's limited based on grid type.  @aliabdolali you work with unstructured grids more than I do, do you know about this?   

@aliabdolali also, all regression tests only use the 0 option : none use 1 or 2 (which is the new feature added here), from what I can find.

Reply:
@JessicaMeixner-NOAA I never used ww3_gint in my unstructured applications. 
Do you suggest having a new regtest with options 1 and 2 ? if yes,  I can add them.